integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/pages/CatalogsPage.java

```diff
@@ -129,9 +136,11 @@ public void setCatalogCommentField(String nameField) {
     }
   }
 
-  public void setHiveCatalogURI(String value) {
+  public void setCatalogPropInput(String key, String value) {
```

Comment:
I think `setCatalogPropInput` better to change to `setCatalogFlixedProp`.
and another function `setCatalogProps` better to change to `setCatalogDynProp` or `setCatalogPropsAt`.